webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,18 @@ function BugForm() {
     }, this));
   };
 
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.baseScrollHeight = this.scrollHeight;
+    })
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      var minRows = 2;
+      this.rows = minRows;
```

Comment:
Can you talk me through your logic of getting this calculation? I'm still trying to understand a few things.

For example, we set `textarea.rows` to 2 here but the other textarea gets set to 3. Why the difference?